wmn-data.json

```diff
@@ -5942,11 +5942,18 @@
       },
       {
         "name" : "Virustotal",
-        "uri_check" : "https://www.virustotal.com/gui/user/{account}",
+        "uri_check" : "https://www.virustotal.com/ui/users/{account}",
+        "pretty_uri" : "https://www.virustotal.com/gui/user/{account}",
+        "headers" : {
+                "User-Agent": "Mozilla/5.0 (X11; Linux x86_64; rv:127.0) Gecko/20100101 Firefox/127.0",

```

Comment:
In checking this detection I noticed the ui/users uri_pretty requires the solving of a captcha. I'm not sure that using this detection will work since the one in the `headers` param is expired. Thoughts?

```
  "error": {
    "code": "RecaptchaRequiredError",
    "message": "Please re-send request with a valid reCAPTCHA response in the \"x-recaptcha-response\" header"
```

Reply:
It seems that Virustotal only checks if the header is present. It doesn't test the x-racaptcha-response at this point (which is X-VT-Anti-Abuse-Header). It passes with all headers values equals to "1".

Reply:
So, I put a string that looks like a token, but it could be anything actually.

Reply:
Thanks for the explanation. My concern is that clients like whatsmyname.app don't pass the header statements but that is not our concern but one for the downstream clients.

Reply:
Yes, as several checks require headers (and maybe more in the future), whatsmyname.app might need to add headers to the client browser requests.